src/app/shared/input-suggestions/validation-suggestions/validation-suggestions.component.spec.ts

```diff
@@ -60,4 +60,30 @@ describe('ValidationSuggestionsComponent', () => {
       expect(comp.onClickSuggestion).toHaveBeenCalledWith(suggestions[clickedIndex].value);
     });
   });
+  describe('can edit input', () => {
+    describe('test input field readonly property when input disable is true', () => {
+      beforeEach(() => {
+        comp.disable = true;
+        fixture.detectChanges();
+      });
+      it('it should be true', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Baryonsoft/dspace-admin/pullRequest?prid=8169620)
```suggestion
      it("it should be true", () => {
```